types/react-google-recaptcha/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React Google Recaptcha 0.10
+// Type definitions for React Google Recaptcha 0.11
```

Comment:
From a quick look, it seems like `onErrored` is not used as part of props in `react-google-recaptcha@0.11`. Can you change the version to reflect the version of `react-google-recaptcha` that you want to describe? Thanks!

Reply:
It's been bumped to v1.0, to coincide with when it was added https://github.com/dozoisch/react-google-recaptcha/commit/2d5b964b801cb54858830abeb45c645c6e83818b

Reply:
@MaxwellBo Thanks for doing that! Unfortunately, given that it was introduced in a new major version, and it seems like v0 is still used, there's another thing I'll need you to do. Can you follow the steps found [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)? Thanks again!